internal/uvm/cpugroup.go

```diff
@@ -10,6 +10,11 @@ import (
 	hcsschema "github.com/Microsoft/hcsshim/internal/hcs/schema2"
 )
 
+// Build that assigning a cpu group on creation of a vm is supported
+const cpuGroupCreateBuild = 20124
+
+var cpuGroupCreateNotSupported = fmt.Errorf("cpu group assignment requires a build of %d or higher", cpuGroupCreateBuild)
```

Comment:
Idiomatic name would be `errCPUGroupCreateNotSupported `.

Reply:
Done